openskill/rate.py

```diff
@@ -156,3 +157,51 @@ def predict_win(teams: List[List[Rating]], **options) -> List[Union[int, float]]
         return probabilities
     else:
         return pairwise_probabilities
+
+
+def predict_draw(teams: List[List[Rating]], **options) -> Union[int, float]:
+    """
+    Predict how likely a match up against teams of one or more agents will draw.
+    This algorithm has a time complexity of O(n!) where 'n' is the number of teams.
+
+    :param teams: A list of two or more teams, where teams are lists of :class:`~openskill.rate.Rating` objects.
+    :return: A :class:`~float` that represents the probability of the teams drawings.
+    """
+    if len(teams) < 2:
+        raise ValueError(f"Expected at least two teams.")
+
+    n = len(teams)
+    total_player_count = sum([len(_) for _ in teams])
+    draw_probability = 1 / n
+    draw_margin = (
+        math.sqrt(total_player_count)
+        * beta(**options)
+        * phi_major_inverse((1 + draw_probability) / 2)
+    )
+
+    pairwise_probabilities = []
+    for pairwise_subset in itertools.permutations(teams, len(teams)):
```

Comment:
```python
itertools.permutations(teams, len(teams))
```
maybe I am missing something, but maybe it could be `itertools.permutations(teams, 2)` to get all of the pairs? `aggregate_multiple` will be different then, but i _think_ you can get the same result.

Reply:
`itertools.permutations(teams, 2)` will only get all permutations of pairs, while the current code works for `n` teams. I must admit, calling the variables `pairwise_probabilities` doesn't aid in understanding the code.